config/config.properties

```diff
@@ -20,162 +20,162 @@
 # Required.
-pravegaservice.zkURL=localhost:2181
+pravegaservice.zk.connect.uri=localhost:2181
 
 # Number of milliseconds to wait between ZooKeeper retries.
 # Valid values: Positive integer.
 # Recommended values: Between 5000 and 10000 (too high will cause failover delays).
-#pravegaservice.zkRetrySleepMs=5000
+#pravegaservice.zk.connect.retries.interval.milliseconds=5000
 
 # Number of times to retry a failed attempt to contact ZooKeeper.
 # Valid values: Positive integer.
 # Recommended values: Between 3 and 5.
-#pravegaservice.zkRetryCount=5
+#pravegaservice.zk.connect.retries.count.max=5
 
 # Session timeout for ZooKeeper.
 # Valid values: Positive integer.
-#pravegaservice.zkSessionTimeoutMs=10000
+#pravegaservice.zk.connect.sessionTimeout.milliseconds=10000
 
 # Enable security for the ZK client.
-#pravegaservice.secureZK=false
+#pravegaservice.zk.connect.security.enable=false
 
 # Location of trust store for secure connection to ZK.
-#pravegaservice.zkTrustStore=
+#pravegaservice.zk.connect.security.tls.trustStore.location=
 
 #Location of password file to access the trust store.
-#pravegaservice.zkTrustStorePasswordPath=
+#pravegaservice.zk.connect.security.tls.trustStore.pwd.location=
 
-# DataLog implementation for Tier 1 storage.
+# DataLog implementation for Durable Data Log Storage.
 # Valid values: BOOKKEEPER, INMEMORY.
 # Default value: BOOKKEEPER
-pravegaservice.dataLogImplementation=BOOKKEEPER
+pravegaservice.dataLog.impl.name=BOOKKEEPER
 
-# Storage implementation for Tier 2 storage.
+# Storage implementation for Long-Term Storage.
 # Valid values: HDFS, FILESYSTEM, EXTENDEDS3, INMEMORY.
 # Default value: HDFS
-# pravegaservice.storageImplementation=HDFS
+# pravegaservice.storage.impl.name=HDFS
 
 # Storage NO-OP Mode: in No-Op mode, user stream segment writing is no-oped; user stream segment reading is not supported.
 # This mode is used to avoid storage interference in testing while still keep the system functioning as usual.
-# NOTE: pravegaservice.storageImplementation is still used to store metadata and system segments, which are required for the functioning of the Pravega Cluster.
+# NOTE: pravegaservice.storage.impl.name is still used to store metadata and system segments, which are required for the functioning of the Pravega Cluster.
 # WARNING: Do NOT set NO-OP Mode to true in production environment! Data loss will happen!
 # Valid values: true, false
 # Default value: false
-#storageextra.storageNoOpMode=false
+#storageextra.noOp.mode.enable=false
 
 # When Storage NO-OP Mode is true, storageWriteNoOpModeLatencyMillis is applied to write operation.
 # It is used to compensate the supposed latency if the storage is in operation, in order to make the test as real as possible.
 # This latency has no effect if storage is not in No-Op mode.
 # Valid values: non-negative integer
 # Default vale: 20 (milliseconds)
-#storageextra.storageWriteNoOpLatencyMillis=20
+#noOp.write.latency.milliseconds=20
```

Comment:
Why this does not have the same structure as the previous property? It refers to the same, so I think the structure should be the same: `storageextra.noOp.write.latency.milliseconds` (unless you replace `storageextra` -> `pravegaservice.storage`, which I think it is easier to understand)

Reply:
This smells like a bug. This value won't be picked up by the code.

Reply:
Yes, that's indeed a bug. I've fixed it now. Thanks for spotting it in [this](https://github.com/pravega/pravega/pull/4713/commits/5481973b7a13aae092d96fa652a9032df33e7074) commit.